packages/@glimmer/core/src/template.ts

```diff
@@ -0,0 +1,37 @@
+import { SerializedTemplateWithLazyBlock, Dict } from '@glimmer/interfaces';
+import { TemplateMeta } from './managers/component/custom';
+
+const TEMPLATE_MAP = new WeakMap<object, SerializedTemplateWithLazyBlock<TemplateMeta>>();
+const getPrototypeOf = Object.getPrototypeOf;
+
+// This is provided by the `babel-plugin-strict-template-precompile` plugin
+export declare function createTemplate(
```

Comment:
From our chat, can we just make this a function that throws and tells people to use the plugin.